ompi/include/mpi.h.in

```diff
@@ -955,8 +955,8 @@ typedef enum {
  * MPIT source ordering
  */
 enum ompi_mpi_t_source_order_t {
-  MPI_T_ORDERED,
-  MPI_T_UNORDERED,
+  MPI_T_SOURCE_ORDERED,
+  MPI_T_SOURCE_UNORDERED,
 };
 
 typedef enum ompi_mpi_t_source_order_t MPI_T_source_order;
```

Comment:
Why isn't this typedef "merged" with the enum declaration, just like the previous `MPI_T_cb_safety`?

```C
typedef enum ompi_mpi_t_source_order_t {
  MPI_T_SOURCE_ORDERED,
  MPI_T_SOURCE_UNORDERED,
} MPI_T_source_order;
```